jenkins-pipelines/longevity-large-partition-2days-reversed-queries.jenkinsfile

```diff
@@ -0,0 +1,12 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    region: 'eu-west-1',
+    test_name: 'longevity_large_partition_test.LargePartitionLongevityTest.test_large_partition_longevity',
+    test_config: '''["test-cases/longevity/longevity-large-partition-2days.yaml", "test-cases/longevity/scylla-bench-stress-order-by-desc.yaml"]'''
```

Comment:
Second files with redefinitions we keep in the "configurations" directory.
"test-cases" must be self-sufficient.

Reply:
fixed